polynote-server/src/main/scala/polynote/server/KernelPublisher.scala

```diff
@@ -107,13 +112,14 @@ class KernelPublisher private (
         case None =>
           val initKernel = for {
               kernel <- createKernel()
+              _      <- handleKernelClosed(kernel).fork
               _      <- kernel.init().provideSomeM(Env.enrichM[BaseEnv with GlobalEnv](cellEnv(-1)))
               _      <- kernelRef.set(Some(kernel))
               _      <- kernel.info() >>= publishStatus.publish1
             } yield kernel
 
           initKernel.tapError {
-            err => closed.succeed(())
```

Comment:
Changed this to just drop the kernel if it fails to init. Not sure why I thought it was a good idea to close the whole KernelPublisher in that case... but this works way better